test/joins/payload-types.ts

```diff
@@ -26,7 +26,39 @@ export interface Config {
     'payload-preferences': PayloadPreference;
     'payload-migrations': PayloadMigration;
   };
-  collectionsSelect?: {
+  collectionsJoins: {
+    posts: {};
```

Comment:
It'd be nicer to not have the keys with empty objects since you can't use it, right?